perf-regression/theta-gpu/gpu-qsub

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#COBALT -A radix-io
+#COBALT -n 2
+#COBALT -t 60
+#COBALT --mode script
+#COBALT -M vchoi@hdfgroup.org
+#COBALT -q full-node
+#COBALT --attr filesystems=home
```

Comment:
Do we want this to run in the home directory? Eagle or Grand might be better on Polaris

Reply:
I think I will add a comment about using whatever scratch filesystem on Thetagpu.  There's right now project space in /lus/theta-fs0/projects/radix-io.